dev/sbt-plugin/src/sbt-test/sbt-plugin/akka-persistence-typed-migration-scala/shopping-cart-akka-persistence-typed/src/main/scala/com/example/shoppingcart/impl/ShoppingCartLoader.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.example.shoppingcart.impl
+
+import com.example.shoppingcart.api.ShoppingCartService
+import com.lightbend.lagom.scaladsl.akka.discovery.AkkaDiscoveryComponents
+import com.lightbend.lagom.scaladsl.devmode.LagomDevModeComponents
+import com.lightbend.lagom.scaladsl.persistence.slick.SlickPersistenceComponents
+import com.lightbend.lagom.scaladsl.server._
+import com.softwaremill.macwire._
+import play.api.db.HikariCPComponents
+import play.api.libs.ws.ahc.AhcWSComponents
+import akka.cluster.sharding.typed.scaladsl.Entity
+import com.lightbend.lagom.scaladsl.playjson.EmptyJsonSerializerRegistry
+
+class ShoppingCartLoader extends LagomApplicationLoader {
+
+  override def load(context: LagomApplicationContext): LagomApplication =
+    new ShoppingCartApplication(context) with AkkaDiscoveryComponents
```

Comment:
Can we stick to using `NoServiceLocator` for `scripted` tests? e.g:

```scala
new ShoppingCartApplication(context) {
      override def serviceLocator = NoServiceLocator
}
```

Then we can also drop the dependency in this sample's `build.sbt` and the `publishLocal` step in the main `build.sbt`.

I mean, all this `scripted` test runs in dev mode, right?